src/client.js

```diff
@@ -28,7 +28,9 @@ IrcClient.prototype._applyDefaultOptions = function(user_options) {
         nick: 'ircbot',
         username: 'ircbot',
         gecos: 'ircbot',
-        encoding: 'utf8'
+        encoding: 'utf8',
+        ping_interval: 30,
+        ping_timeout: 120,
```

Comment:
So how is it going to work when ping interval is smaller than the timeout? Will it keep sending pings and resetting the timeout timer?